recipes/zxing-cpp/all/conanfile.py

```diff
@@ -66,13 +65,11 @@ def validate(self):
         min_version = self._compiler_cpp14_support.get(str(self.settings.compiler))
         if min_version and tools.Version(self.settings.compiler.version) < min_version:
             raise ConanInvalidConfiguration(
-                "This compiler is too old. This library needs a compiler with c++14 support"
-            )
-        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) >= "11":
-            raise ConanInvalidConfiguration(
-                "zxing-cpp doesn't support gcc >= 11. Contributions are "
-                "welcome if you want to fix the build."
+                "This compiler is too old. {} needs a compiler with c++14 support".format(self.name)
             )
+        # FIXME: This is a workaround for "The system cannot execute the specified program."
+        if tools.Version(self.version) == "1.3.0" and is_msvc_static_runtime(self) and self.settings.build_type == "Debug":
+            raise ConanInvalidConfiguration("{}/{} doesn't support MT + Debug.".format(self.name, self.version))
```

Comment:
It is this combination wou want to exclude from the build matrix?
Can you reproduce the problem locally? Is it VS2017-only, or does it happen for all VC's?

The error you're trying to work around ("The system cannot execute the specified program.`) looks weird.
This log? https://github.com/conan-io/conan-center-index/pull/11210#issuecomment-1157258162